kafkaApplicationSimulator/kafkaq-config.js

```diff
@@ -0,0 +1,5 @@
+const kafkaq = require('../kafkaq-monitor/index.js');
```

Comment:
what is this doing? did we not have/need this before?

Reply:
I have no idea how that file got here... I deleted the branch it was on. I will delete the file now. Good catch!